Lib/sqlite3/test/userfunctions.py

```diff
@@ -475,6 +506,13 @@ def authorizer_cb(action, arg1, arg2, dbname, source):
         return sqlite.SQLITE_OK
 
 
+class AuthorizerUnhashable(AuthorizerTests):
+    def setUp(self):
+        super().setUp()
+        with self.assertRaisesRegex(TypeError, "unhashable type"):
```

Comment:
Is it possible to test not in `setUp()`?

Reply:
It can be reworked in this way:
```
class AuthorizerUnhashable(AuthorizerTests):
    def test_table_access(self):
        with self.assertRaisesRegex(TypeError, "unhashable type"):
            self.con.set_authorizer(UnhashableFunc(sqlite.SQLITE_OK))
        super().test_table_access()

    def test_column_access(self):
        with self.assertRaisesRegex(TypeError, "unhashable type"):
            self.con.set_authorizer(UnhashableFunc(sqlite.SQLITE_OK))
        super().test_column_access()
```
Does it look better?